webui.py

```diff
@@ -1077,8 +1143,7 @@ def img2img(prompt: str, image_editor_mode: str, init_info, mask_mode: str, mask
         keep_mask = False
 
     assert 0. <= denoising_strength <= 1., 'can only work with strength in [0.0, 1.0]'
-    t_enc = int(denoising_strength * ddim_steps)

```

Comment:
why have you moved this? pretty sure this will break non k-diffusion samplers which need t_enc too